cmd/horcrux/cmd/config.go

```diff
@@ -33,12 +33,12 @@ func configCmd() *cobra.Command {
 
 func initCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "init [chain-nodes]",
+		Use:     "init",
```

Comment:
Instead of all these flags, I wonder if `init` generates an example that the user must then hand-edit? 

That way, we can simply delete all these flags and make this subcommand simpler. 

For our own deployments, we likely won't use this subcommand anymore. Rather, we'll create the config file as a configmap and mount it into the container.

Reply:
Maybe a `--bare` flag to generate an empty config (or disable validation) would be helpful? Right now this command enforces that the final config is valid.

Reply:
Nah, I'd keep it as-is then instead of adding more complexity. It was just a thought, not a blocker.